trading_calendars/exchange_calendar_xnys.py

```diff
@@ -160,6 +160,7 @@ def special_closes_adhoc(self):
                         '1999-12-31',
                         '2003-12-26',
                         '2013-07-03',
+                        '2019-07-03'
```

Comment:
Purely stylistic, but want to add a trailing comma, so we avoid extra noise in subsequent PRs?

Reply:
Fixed